Main.java

```diff
@@ -9,7 +11,19 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //ask the user for the tempurature in celsuis
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius.");
+
+    //declare and initialize variable for tempurature
+    int tempCelsuis = input.nextInt();
+    
+    //declare and calculate degrees farenheit
+    int tempFarenheit = (tempCelsuis*9)/5 + 35;
```

Comment:
Good work with the program. The reason it is failing is because the formula is incorrect. You should be adding 32 instead of 35. Otherwise, it looks good!